[2022-09-25T12:32:20.020+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: migrate_data.create_table manual__2022-09-25T12:32:18.463521+00:00 [queued]>
[2022-09-25T12:32:20.031+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: migrate_data.create_table manual__2022-09-25T12:32:18.463521+00:00 [queued]>
[2022-09-25T12:32:20.031+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-09-25T12:32:20.032+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 3
[2022-09-25T12:32:20.032+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-09-25T12:32:20.044+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2022-09-25 12:32:18.463521+00:00
[2022-09-25T12:32:20.049+0000] {standard_task_runner.py:54} INFO - Started process 79 to run task
[2022-09-25T12:32:20.051+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'migrate_data', 'create_table', 'manual__2022-09-25T12:32:18.463521+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/migrate.py', '--cfg-path', '/tmp/tmpzwhir_1j']
[2022-09-25T12:32:20.051+0000] {standard_task_runner.py:83} INFO - Job 8: Subtask create_table
[2022-09-25T12:32:20.052+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/***/dags/migrate.py
[2022-09-25T12:32:20.512+0000] {task_command.py:384} INFO - Running <TaskInstance: migrate_data.create_table manual__2022-09-25T12:32:18.463521+00:00 [running]> on host c138e22c1d18
[2022-09-25T12:32:20.577+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=isaaclucky88@gmail.com
AIRFLOW_CTX_DAG_OWNER=yishaktadele
AIRFLOW_CTX_DAG_ID=migrate_data
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2022-09-25T12:32:18.463521+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-09-25T12:32:18.463521+00:00
[2022-09-25T12:32:20.584+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2022-09-25T12:32:20.587+0000] {sql.py:315} INFO - Running statement: -- CREATE DATABASE IF NOT EXISTS `city_traffic`;
CREATE TABLE IF NOT EXISTS  traffic_data (
   track_id INT PRIMARY KEY,
   type varchar(20),
   traveled_d FLOAT,
   avg_speed FLOAT,
   lat FLOAT,
   lon FLOAT,
   speed FLOAT, 
   lon_acc FLOAT,
   lat_acc FLOAT,
   time FLOAT
);, parameters: None
[2022-09-25T12:32:20.589+0000] {postgres.py:94} INFO - NOTICE:  relation "traffic_data" already exists, skipping

[2022-09-25T12:32:20.596+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=migrate_data, task_id=create_table, execution_date=20220925T123218, start_date=20220925T123220, end_date=20220925T123220
[2022-09-25T12:32:20.624+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2022-09-25T12:32:20.652+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
